tests/utils_tests/test_html.py

```diff
@@ -75,6 +76,26 @@ def test_format_html_no_params(self):
             name = "Adam"
             self.assertEqual(format_html(f"<i>{name}</i>"), "<i>Adam</i>")
 
+    def test_format_html_join_with_positional_arguments(self):
+        self.assertEqual(
+            format_html_join(
+                "\n",
+                "<li>Id: {}</li><li>Text: {}</li>",
+                [(1, "Text1"), (2, "Text2")],
+            ),
+            "<li>Id: 1</li><li>Text: Text1</li>\n<li>Id: 2</li><li>Text: Text2</li>",
```

Comment:
Making the test use a simpler, more realistic template and data:

```suggestion
                "<li>{id}) {text}</li>",
                [(1, "Emma"), (2, "Matilda")],
            ),
            "<li>1) Emma</li>\n<li>2) Matilda</li>",
```

(It would rarely make sense to put ids and text values in separate `<li>`  list items.)